docs/source/repair/index.rst

```diff
@@ -51,20 +50,43 @@ In a benchmark we ran 9 Scylla 2020.1 nodes on AWS i3.2xlarge machines under 50%
 
 By default Scylla Manager runs repairs with full parallelism, you can change that using :ref:`sctool repair --parallel flag <sctool-repair>`.
 
+Maximal effective parallelism
+=============================
+
+Max parallelism is limited by the ``each node can take part in at most one Scylla repair job at any given moment`` constraint.
```

Comment:
I'd reorganize and reword the description for the sake of clarity:

> Max parallelism is determined by:
> * the constraint that each node can only take part in one ScyllaDB repair job at any given moment.
> * ScyllaDB repair job targeting the full replica set of the repaired token range.
> 
> For example, let's assume a cluster with 2 datacenters, 5 nodes each. When you repair the keyspace ``my_keyspace with replication = {'class': 'NetworkTopologyStrategy', 'dc1': 2, 'dc2': 3}``, max parallelism is equal to ``1``, because each ScyllaDB repair job targets a full replica set of the repaired token range. Every replica set consists of 2 nodes from ``dc1`` and 3 nodes from ``dc2``, so it's impossible to schedule 2 repair jobs to run simultaneously (``dc2`` lacks one more node for it to be possible). 

(``dc2`` lacks one more node for it to be possible) - is it``dc2``?

Also, please add an empty space before
> Repair is performed table by table and keyspace by keyspace,
> so max effective parallelism might change depending on which 
> keyspace is being repaired.

Otherwise, the content displays on the same line (and it shouldn't, as it's not part of the example).

Reply:
> (dc2 lacks one more node for it to be possible) - is itdc2?

Yes, because `dc2` has 5 nodes and rf=3, so it's not possible to run two repair jobs without using some node in both of them (and SM does not allow for it to happen). If `dc2` had 1 more node, it would be possible.

Reply:
Just making sure I understand correctly. Thanks.